# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libcap/cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Helper tools must be built by the host-compiler ...

  - Rene Rebe <rene@exactcode.de>

--- ./libcap/Makefile.vanilla	2007-02-25 12:17:01.132916308 +0100
+++ ./libcap/Makefile	2007-02-25 12:17:13.061596085 +0100
@@ -45,7 +45,7 @@
 all: $(MINLIBNAME)
 
 _makenames: _makenames.c cap_names.sed
-	$(CC) $(CFLAGS) $(LDFLAGS) $< -o $@
+	cc $(CFLAGS) $(LDFLAGS) $< -o $@
 
 cap_names.h: _makenames
 	./_makenames > cap_names.h
